langstream-runtime/langstream-runtime-impl/src/main/java/ai/langstream/runtime/agent/AgentRunner.java

```diff
@@ -347,6 +348,7 @@ private static void runJavaAgent(
                     AgentContext agentContext =
                             new SimpleAgentContext(
                                     agentId,
+                                    configuration.agent(),
```

Comment:
this is wrong, in case of the "composite agent" this doesn't make sense, or maybe it is the id of the first agent in the group